### PR TITLE
[EOC-574] Landing page update

### PR DESCRIPTION
**Pull request by adamklepacz**
I think it is hard to understand something from the screenshots so I recommend you guys, pull my branch and just open index.html file locally.

Zrzut ekranu 2019-10-28 o 15 52 17
Zrzut ekranu 2019-10-28 o 15 52 11
Zrzut ekranu 2019-10-28 o 13 17 13
Zrzut ekranu 2019-10-28 o 13 16 55
Zrzut ekranu 2019-10-28 o 13 16 18
Zrzut ekranu 2019-10-28 o 13 15 59
Zrzut ekranu 2019-10-28 o 13 15 51
Zrzut ekranu 2019-10-28 o 13 15 35
Zrzut ekranu 2019-10-28 o 13 15 16
Zrzut ekranu 2019-10-28 o 13 14 49
Zrzut ekranu 2019-10-28 o 13 14 38
Zrzut ekranu 2019-10-28 o 13 14 33
Zrzut ekranu 2019-10-28 o 13 14 23
Zrzut ekranu 2019-10-28 o 13 14 16
Zrzut ekranu 2019-10-28 o 12 13 21
Zrzut ekranu 2019-10-28 o 12 13 12
Zrzut ekranu 2019-10-28 o 12 13 07

### CODE REVIEW

File: landing/index.html [hunk 1]
```diff
@@ -4,52 +4,29 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags
-    -->
-    <meta
-      name="description"
-      content="End Of Coffee - Track your office inventory"
-    />
+    <meta name="description" content="End Of Coffee" />
```
Comment (alemikolo) [inline landing/index.html]: proposal: the description is important in SEO and should be more descriptive: "End of coffee helps you track all the stuff you care about!"

File: landing/index.html [hunk 2]
```diff
@@ -4,52 +4,29 @@
...
     <meta name="author" content="Sandstream Development" />
-
-    <!-- Title -->
-    <title>End Of Coffee - Track Your office inventory</title>
-
-    <!-- Necessary CSS Files -->
-    <!--
-      <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700" rel="stylesheet">	 Google Font
-    -->
+    <title>End Of Coffee</title>
```
Comment (alemikolo) [inline landing/index.html]: proposal: add more descriptive info?

Comment (adamklepacz) [reply landing/index.html]: suggest something

Comment (alemikolo) [reply landing/index.html]: I think about something similar to description:
**title:**
End of coffee - an app that helps you track all the stuff you care about!" 

This text is better for the title. Description can be and should be a little longer:
**Description:**
End Of Coffe - real-time app for planning and managing your staff. Use it individually or with friends. Create and edit lists, manage cohorts on any device and regardless where you are.

you write it much better than me

File: landing/index.html [hunk 3]
```diff
@@ -250,50 +202,49 @@ <h2>Sign up for a newsletter</h2>
...
               </p>
-
               <div class="nomber-text wow fadeInUp">
                 <div class="nomber-text-row">
                   <div class="nomber-text-row-inner">
                     <span>1. Sacks</span>
                     <p>
-                      Create your personal sacks to manage things that you don't
-                      want to forget about.
+                      Create personal, share or limited sacks. It depends on
```
Comment (marekrozmus) [inline landing/index.html]: shared

File: landing/index.html [hunk 4]
```diff
@@ -250,50 +202,49 @@ <h2>Sign up for a newsletter</h2>
...
                   <div class="clear"></div>
                 </div>
-
                 <div class="nomber-text-row">
                   <div class="nomber-text-row-inner">
                     <span>2. Cohorts </span>
                     <p>
-                      Create and manage your cohorts. Add members, sacks and set
-                      up permissions. Collaborate with others!
+                      Create and manage your cohort to easily share sacks and
```
Comment (marekrozmus) [inline landing/index.html]: your cohorts

**Pull request merged**